tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,35 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"testing"
+
+	testutils "k8s.io/cloud-provider-azure/tests/e2e/utils"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+
+	_ "k8s.io/cloud-provider-azure/tests/e2e/autoscaling"
+	_ "k8s.io/cloud-provider-azure/tests/e2e/network"
+)
+
+func TestAzureTest(t *testing.T) {
+	RegisterFailHandler(Fail)
+	DNSPrefix := testutils.ExtractDNSPrefix()
+	reportDir := DNSPrefix + "/report/"
```

Comment:
can just hard code to '_report', and make it a parameter later, do not retrieve testutils.ExtractDNSPrefix